contrib/babelfishpg_tsql/src/pltsql_utils.c

```diff
@@ -201,7 +202,33 @@ pltsql_createFunction(ParseState *pstate, PlannedStmt *pstmt, const char *queryS
 		return false;
 	}
 	else
-	{	
+	{
+		/* Do not check for duplicate function if REPLACE flag is set and during dump and restore. */
+		if (!stmt->replace && !babelfish_dump_restore)
```

Comment:
we should add test cases with different variation
same name functions, procedures, function/procedure, table/function/procedure
With some variant of arguments: no args, one arg, same number of args, same type of args, same number but different type and so on
create/replace

Reply:
Added tests